quickstarts/blazor-webassembly/config.yml

```
@@ -0,0 +1,53 @@
+id: d47d1298-fe2a-437b-91f1-c0316bd274d7
+
+
+slug: blazor-webassembly
```

let's not change this slug. If we change it, that will then change the URL for this quickstart on newrelic.com/instant-observability. 
```suggestion
slug: blazor-web-assembly
```

---

Hi @aswanson-nr ,
I changed the slug name as per your comment. Kindly review it.

Thank you...